pkg/autodiscovery/autoconfig.go

```diff
@@ -662,15 +656,15 @@ func (ac *AutoConfig) processNewService(ctx context.Context, svc listeners.Servi
 
 // processDelService takes a service, stops its associated checks, and updates the cache
 func (ac *AutoConfig) processDelService(svc listeners.Service) {
-	ac.store.removeServiceForEntity(svc.GetEntity())
-	removedConfigs := ac.store.removeConfigsForService(svc.GetEntity())
+	ac.store.removeServiceForEntity(svc.GetServiceID())
+	removedConfigs := ac.store.removeConfigsForService(svc.GetServiceID())
 	ac.processRemovedConfigs(removedConfigs)
 	ac.store.removeTagsHashForService(svc.GetTaggerEntity())
 	// FIXME: unschedule remove services as well
 	ac.unschedule([]integration.Config{
 		{
 			LogsConfig:      integration.Data{},
-			Entity:          svc.GetEntity(),
+			ServiceID:       svc.GetServiceID(),
```

Comment:
Thanks for looking!  Calling these "entities" is what initially confused me and led to this change.  The things this value labels are of type `pkg/autodiscovery/listeners.Servce`, so I think calling it a "Service ID" makes more sense than "Entity ID" (since there is no struct named "Entity" in the AD package).   WDYT?